pkgs/development/tools/misc/packr/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, fetchFromGitHub, buildGoModule }:
+
+let
+  owner = "gobuffalo";
+  pname = "packr";
+  version = "2.1.0";
+in
+buildGoModule rec {
+  name = "${pname}-${version}";
```

Comment:
`buildGoModule` does support pname, so you may skip setting the `name` all together.